src/helpers/issue.ts

```diff
@@ -263,6 +263,22 @@ export const getPullRequests = async (context: Context, state: "open" | "closed"
   }
 };
 
+export const getPullRequestReviews = async (context: Context, pull_number: number) => {
+  const logger = getLogger();
+  const payload = context.payload as Payload;
+  try {
+    const { data: reviews } = await context.octokit.rest.pulls.listReviews({
```

Comment:
This [API](https://docs.github.com/en/rest/pulls/reviews?apiVersion=2022-11-28#list-reviews-for-a-pull-request) returns paginated results. You sure that all review are returned here?

Reply:
According to the official Octokit documentation, the listReviews() function returns all of the reviews for a pull request by default, without any pagination.

The purpose of using the listReviews() function is not to retrieve all the reviews, but merely to check whether a specific pull request has any reviews or not.  And I think its fine.

Reply:
Could you share Octokit documentation URL you are referring to ?

Reply:
@wannacfuture ould you share Octokit documentation URL you are referring to ?